# This message holds supply status for each joint:
# - name - names of joints
# - state - supply state (current, voltage) of joints
#
# All arrays should have same size
#
# Note: AR600 using joints' numbers as names

string[] names
SupplyState[] states